k4MarlinWrapper/examples/event_display.py

```diff
@@ -197,6 +211,7 @@
                               "PandoraPFOCollection", "0", "3", "7",
                               "JetOut", "0", "0", "3"],
                           "DrawSurfaces": ["true"],
+                          # "DrawSurfaces": ["false"],
```

Comment:
Do I understand correctly, that we had this option in here twice with opposite values before?